next.config.js

```diff
@@ -11,6 +11,8 @@ const nextConfig = {
     },
   },
   async headers() {
+    const isDev = process.env.NEXT_PUBLIC_ORIGIN !== 'https://ironfish.network/';
```

Comment:
suggestion: I think the more common way of checking for this is `proecss.env.NODE_ENV === "development"`